### PR TITLE
Start refactoring build_deploy.sh to add fleet management builds

### PR DESCRIPTION
**Pull request by thom-at-redhat**
# Description

Start of moving Travis fleet management container builds to build_deploy.sh

Fixes # (issue)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [X] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

**Review by de1987 — APPROVED** [review]
lgtm!

File: build_deploy.sh [hunk 1]
```diff
@@ -2,45 +2,115 @@
...
     exit 1
 fi
 
-if [[ -z "${RH_REGISTRY_USER}" || -z "${RH_REGISTRY_TOKEN}" ]]; then
-    echo "RH_REGISTRY_USER and RH_REGISTRY_TOKEN  must be set"
-    exit 1
-fi
+podman login -u="${QUAY_USER}" -p="${QUAY_TOKEN}" quay.io
 
+#if [[ -z "${RH_REGISTRY_USER}" || -z "${RH_REGISTRY_TOKEN}" ]]; then
```
Comment (acosferreira) [inline build_deploy.sh:17]: sholud we remove those commented lines?

File: build_deploy.sh [hunk 2]
```diff
@@ -2,45 +2,115 @@
...
+FLEET_MANAGEMENT_LIBFDO_CONTAINERFILE="test-container/Dockerfile"
+# shellcheck disable=SC2034
+readonly FLEET_MANAGEMENT_LIBFDO_CONTAINERFILE
+
+declare FLEET_MANAGEMENT_EDGE_API_CONTAINERFILE
+FLEET_MANAGEMENT_EDGE_API_CONTAINERFILE="Dockerfile"
+# shellcheck disable=SC2034
+readonly FLEET_MANAGEMENT_EDGE_API_CONTAINERFILE
+
+# Travis before_install
```
Comment (acosferreira) [inline build_deploy.sh:80]: should we remove the commented lines for travis?

File: build_deploy.sh (hunk 1 above)
Comment (thom-at-redhat) [reply build_deploy.sh:17]: I was leaving them in temporarily.  They should be removed in the next PR which updates pr_check.sh

File: build_deploy.sh (hunk 2 above)
Comment (thom-at-redhat) [reply build_deploy.sh:80]: I was leaving them in temporarily.  They should be removed in the next PR which updates pr_check.sh (Demetrius asked the same question)

Comment (thom-at-redhat) [reply build_deploy.sh:78]: I added more logic to make it clearer

**Comment by omaciel** [comment]
Added a `do not merge` label as per @thom-at-redhat request (via text) :)

**Comment by thom-at-redhat** [comment]
New approach soon

**Pull request closed without merging**